modules/nested-accordion/widgets/nested-accordion.php

```diff
@@ -636,6 +648,29 @@ private function add_header_color_style( $state, $context ) {
 			]
 		);
 
+		if ( 'title' === $context ) {
+			$this->add_group_control(
+				Group_Control_Text_Shadow::get_type(),
+				[
+					'name' => $context . '-' . $state . '-text-shadow',
```

Comment:
```suggestion
					'name' => $context . '_' . $state . '-text-shadow',
```
Maybe?

Reply:
@louiswol94 These match with the variable naming in SCSS
![image](https://github.com/elementor/elementor/assets/130742176/4202c2ce-db5d-4a2b-9292-474e2b4b4a45)

![image](https://github.com/elementor/elementor/assets/130742176/bfd9f0e5-80ad-462b-a757-abcabdf0ef88)